slackbridge/config.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "nickname": "osh-chatbot",
```

Comment:
...then include the vars here like:

```
[
  {
    "nickname": "$NICKNAME",
    "server": "$SERVER",
    "token": "$TOKEN",
    "autoSendCommands": [
        ["MODE", "$MODE", "+i"],
        ["AUTH", "$AUTH", "$PASSWORD"]
    ],
    "channelMapping": {
      "#openstack-helm": "#openstack-helm"
    }
  }
]
```

don't take that to be perfect...just merely an example.